core/taskengine/doc.go

```diff
@@ -24,5 +24,10 @@ Storage can also be inspect with telnet:
 	telnet /tmp/ap.sock
 
 Then issue `get <ket>` or `list <prefix>` or `list *` to inspect current keys in the storage.
+
+**Secret Storage**
+- currently org_id will always be _ because we haven't implemented it yet
+- workflow_id will also be _ so we can search be prefix
```

Comment:
I don’t quite get this line, `- workflow_id will also be _ so we can search be prefix`

Does this mean the search is not done at the workflow level, but only user level? I’m fine with that, but let’s clarify this comment.

Reply:
oh I see. Makes sense 👌